### PR TITLE
toonify any image using open cv

### PR DESCRIPTION
**Pull request by Tejas1510**
### Your checklist for this pull request
:warning: Please review the [guidelines for contributing](https://github.com/sanscript-tech/hacking-tools-scripts/blob/main/CONTRIBUTING.md) to this repository.

Parent issue: #294 

### Type of change

**Select all that applies**:
* [x] New feature (non-breaking change which adds functionality)
* [ ] Bug fix (non-breaking change which fixes an issue)
* [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
* [ ] This change requires a documentation update

### Checklist:

* [x] My code follows the style guidelines(Clean Code) of this project.
* [x] I have performed a self-review of my own code.
* [x] I have used consistent code conventions and clear function/method/variable names.
* [x] I have commented on my code, particularly in hard-to-understand areas.
* [x] I have created a helpful and easy to understand README.md which contains description, requirements and use case.
* [x] I have added relevent screenshots/images if necessary in a seperate folder.
* [x] I have added my work in respective language folder.
* [x] My changes generate no new warnings.

### CODE REVIEW

File: Python/ToonifyImage/toonify.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import requests
+import shutil # to save image locally
+imagePath=input("Please enter the path of image to toonify : ")
+api_key=input("Please enter your api key")
+r = requests.post(
```
Comment (yashaswibiyahut) [inline Python/ToonifyImage/toonify.py:8]: Add proper comments in your code

**Review by yashaswibiyahut — CHANGES_REQUESTED** [review]

**Comment by yashaswibiyahut** [comment]
Change the name of your PR with suitable name.

**Comment by Tejas1510** [comment]
@yashaswibiyahut I have done the changes requested

**Comment by yashaswibiyahut** [comment]
simply give the name same as the parent name.
**toonify any image using open cv**

**Review by yashaswibiyahut — APPROVED** [review]

**Pull request merged**